daemon/api_test.go

```diff
@@ -52,6 +52,7 @@ import (
 	"github.com/snapcore/snapd/asserts/assertstest"
 	"github.com/snapcore/snapd/asserts/sysdb"
 	"github.com/snapcore/snapd/client"
+	"github.com/snapcore/snapd/cmd"
```

Comment:
This is nothing new so not relevant for this PR - but it does feel slightly odd to me that `apinfo.go` (or now `snapinfo.go` lifes under `github.com/snapcore/snapd/cmd`). Seems like something we may want to look at at some point.

Reply:
It's there until it grows up a little more at which point we might know better where to put it. It was the result of a long conversation with @pedronis about exactly where to put it...